crates/api_common/src/utils.rs

```diff
@@ -915,30 +915,48 @@ async fn proxy_image_link_internal(
 
 /// Rewrite a link to go through `/api/v4/image_proxy` endpoint. This is only for remote urls and
 /// if image_proxy setting is enabled.
-pub async fn proxy_image_link(link: Url, context: &LemmyContext) -> LemmyResult<DbUrl> {
-  proxy_image_link_internal(link, context.settings().pictrs()?.image_mode, context).await
+pub async fn proxy_image_link(
+  link: Url,
+  is_thumbnail: bool,
+  context: &LemmyContext,
+) -> LemmyResult<DbUrl> {
+  proxy_image_link_internal(
+    link,
+    context.settings().pictrs()?.image_mode,
+    is_thumbnail,
+    context,
+  )
+  .await
 }
 
 pub async fn proxy_image_link_opt_apub(
   link: Option<Url>,
   context: &LemmyContext,
 ) -> LemmyResult<Option<DbUrl>> {
   if let Some(l) = link {
-    proxy_image_link(l, context).await.map(Some)
+    proxy_image_link(l, false, context).await.map(Some)
```

Comment:
Why is this one always false?

Reply:
This function is only called for icons, banners and avatars but not for any thumbnails.